python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -149,24 +139,39 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
                   .join(", "),
             },
             {
-              label: "Pending",
+              label: "Number of Pending Tasks",
               value: actor.taskQueueLength.toLocaleString(),
+              tooltip:
+                "The number of tasks that are currently pending to execute on this actor. If this number " +
+                "remains consistently high, it can indicate that this actor is a bottle-neck in your program.",
```

Comment:
We can also have a tooltip somewhere that can link users to the dashboard document?

Reply:
Great idea!

Reply:
In general, pointing to relevant documentation pages wherever possible would be great. For example, pointing to the `ray memory` docs for number of object iDs in scope would be good

Reply:
I added links to the documentation where relevant as recommended.